concourse/update-content/script.bash

```diff
@@ -36,22 +36,10 @@ for book_id in $book_ids; do
   # are available for the recipe code version REX supports
   desired_version=$(sort --version-sort -r <<< "$approved_versions" | head -n 1)
 
-  current_version=$(jq -r --arg uuid "$book_id" '.[$uuid].defaultVersion' < src/config.books.json)
-
   # approved book format has a "1." on the front of every version
   desired_version=${desired_version:2}
 
-  if [ "$desired_version" == "$current_version" ]; then
-    echo "$book_id alredy at desired version."
-    continue
```

Comment:
in the case that there is no update, we want to continue to not open a pull request... will github reject opening a pull request that has no changes? or do we need to add logic after your new script that noops if there are no staged changes?

Reply:
> will github reject opening a pull request that has no changes?

there is no way to do this from UI, but it doesn't matter because we would still create and push branch so we have to detect that nothing has changed